superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -1044,14 +1051,50 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     setIsHidden(false);
   }
 
+  const SAVE_TEXT = t('Save');
```

Comment:
@lyndsiWilliams are we going to move these into a `TRANSLATIONS` object?

Reply:
Definitely! Done in [`this commit`](https://github.com/apache/superset/pull/22066/commits/c74039460331569ae9d29c6398fd2dd937a3971b).